src/ontology/reports/sync-synonym/icd11foundation.synonyms.added.robot.tsv

```diff
@@ -148,7 +148,7 @@ MONDO:0000680	astereognosia	oio:hasExactSynonym		tactile agnosia			icd11.foundat
 MONDO:0000680	astereognosia	oio:hasExactSynonym		tactile amnesia			icd11.foundation:756147896	Astereognosia					tactile amnesia	icd11.foundation:756147896										
 MONDO:0000687	diffuse alopecia areata	oio:hasExactSynonym		Diffuse alopecia areata			icd11.foundation:666274267	Diffuse alopecia areata					Diffuse alopecia areata	icd11.foundation:666274267										
 MONDO:0000690	body dysmorphic disorder	oio:hasExactSynonym		Body dysmorphic disorder			icd11.foundation:731724655	Body dysmorphic disorder					Body dysmorphic disorder	icd11.foundation:731724655										
-MONDO:0000698	gamma-amino butyric acid metabolism disorder	oio:hasExactSynonym		Disorders of GABA - [gamma aminobutyric acid]  metabolism			icd11.foundation:1811657969	Disorders of gamma aminobutyric acid metabolism					Disorders of GABA - [gamma aminobutyric acid]  metabolism	icd11.foundation:1811657969										
+MONDO:0000698	gamma-amino butyric acid metabolism disorder	oio:hasExactSynonym		Disorders of GABA - [gamma aminobutyric acid] metabolism			icd11.foundation:1811657969	Disorders of gamma aminobutyric acid metabolism					Disorders of GABA - [gamma aminobutyric acid] metabolism	icd11.foundation:1811657969										
```

Comment:
## Modified `-added` entry
An extra space was removed from _Disorders of GABA - [gamma aminobutyric acid]  metabolism_. This synonym doesn't yet exist in Mondo. If it did, this would be an example of an `-added` entry being moved into `-confirmed`. But since it doesn't exist in Mondo, the result is simply a modification of this row in-situ; the redundant space is removed.

There appear to be many such cases of this in `icd11foundation`.